src/lib/components/table/table-column-settings/dialog/table-column-settings-dialog.component.html

```diff
@@ -0,0 +1,17 @@
+<h2 mat-dialog-title>{{ 'columnSettings.overlayTitle' | translate:_lang}}</h2>
+<mat-dialog-content>
+    <mat-selection-list [(ngModel)]="_selectedColumns">
+        <mat-list-option *ngFor="let column of _columns" [value]="column.value">
+            {{column.caption}}
+        </mat-list-option>
+    </mat-selection-list>
+</mat-dialog-content>
+<mat-dialog-actions>
+    <terra-button [mat-dialog-close]="true"
+                  [inputCaption]="'columnSettings.confirm' | translate:_lang"
+                  [outputClicked]="_customize()">
+    </terra-button>
+    <terra-button [inputCaption]="'columnSettings.cancel' | translate:_lang"
+                  (click)="onCancel()">
```

Comment:
```suggestion
                  mat-dialog-close>
```

Reply:
würde das nicht dafür sorgen das die selected columns leer wären?

Reply:
obwohl.. sollte sich in beiden fällen gleich verhalten

Reply:
im beispiel der selection list wurde es so gemacht wie es momentan implementiert ist aber ich hab ehrlich gesagt keine ahnung warum

Reply:
Ja. Das musst du halt im afterClosed abfangen. Bei deinem `(click)="onCancel()"` aber auch.
Die veränderten `selectedColumns` möchtest du ja nicht zurückgeben. Sonst würden ja auch beim Abbrechen, die Änderungen übernommen werden.